docs/modules/commons/pages/table-transformers.adoc

```diff
@@ -516,3 +516,96 @@ Examples:
 |value |1    |
 |value2|0    |
 ----
+
+== INNER_JOIN
+
+`INNER_JOIN` transformer combines rows from two tables whenever there are matching values between the columns.
+
+[cols="1,3", options="header"]
+|===
+|Parameter
+|Description
+
+|`leftTableJoinColumn`
+|the column name for matching in the first (left) table
+
+|`rightTableJoinColumn`
+|the column name for matching in the second (right) table
```

Comment:
let's do not confuse users: use only left and right, but not first and second

Reply:
done